project/pom.xml

```diff
@@ -217,12 +217,12 @@
   </developers>
 
   <properties>
-    <maven.compile.source>1.7</maven.compile.source>
-    <maven.compile.target>1.7</maven.compile.target>
-    <jdk.version>1.7</jdk.version>
+    <maven.compile.source>1.8</maven.compile.source>
+    <maven.compile.target>1.8</maven.compile.target>
+    <jdk.version>1.8</jdk.version>
     <maven.compile.deprecation>true</maven.compile.deprecation>
     <maven.site.url.base>gitsite:git@github.com/jclouds/jclouds-maven-site.git</maven.site.url.base>
-    <guava.version>18.0</guava.version>
+    <guava.version>22.0</guava.version>
```

Comment:
We could support 21.0 with some workarounds for `SimpleTimeLimiter` although this doesn't seem worth the effort.